### ISSUE TITLE
Using other characters besides greater/less than for messages breaks linking

### ISSUE THREAD

**Issue by Naththo** [hexchat/hexchat#1316]
Using things like this for "Channel Message" and "Your Channel" seems to break the linking (clicking their name directly in the main chat window).

`($1) $2`
`{$1} $2`

I know that the characters in the second example could be used in nicks, which is why I thought at first would be the problem, but not even () works.

**Comment by TingPing** [comment]
Odd considering () works around other types. @RichardHitt might know why that is.

**Comment by RichardHitt** [comment]
In xtext.c there's a macro called is_del(char) which evaluates to TRUE 
if the character is thought to be a delimiter.  Historically besides the 
usual white-space and newline sorts of character it included left and 
right parentheses and those arrow glyphs.  These days the parenthesis 
characters have been removed, probably to enable Wikipedia-type URLs 
which may contain parens.  I don't know the effect (yet) of removing 
also '<' and '>' but I'll give it a try and see what blows up.  I have a 
hunch this is the root of the problem.

Richard Hitt

**Comment by RichardHitt** [comment]
So I tried eliminating '<' and '>' and as a result, mousing over the 
header nick of a message does not detect the nick, though mousing over a 
nick embedded within a message does detect the nick. Not good.  I'll see 
if I can think up a reasonable compromise and test it.

Richard

**Comment by Naththo** [comment]
Thanks for letting us know, Richard.

**Comment by RichardHitt** [comment]
Naththo, with my suggested fix you would need to do more than just code "($1) $2"; you'd need to stick in a pair of %H's as well, like this:  "(%H%H$1%H%H)$t$2".  But at least you'd get on the text window what you wanted.  Is this all right with you?

**Comment by Naththo** [comment]
Yeah, that's fine. As long as it works around nicks then that's fine, no matter what it takes.
